### ISSUE TITLE
[suggestion] Améliorer la gestion des plugins

### ISSUE THREAD

**Issue by Tabrisange** [LeedRSS/Leed#465]
Voilà ce que j'ai dans mon fichier plugins.states.json: 

{"\/data\/www\/rss.neurozone.fr\/plugins\/DeleteTheCache\/DeleteTheCache.plugin.disabled.php":true,"\/data\/www\/rss.neurozone.fr\/plugins\/smartfeedsupdate\/smartfeedsupdate.plugin.php":false,"\/data\/www\/rss.neurozone.fr\/plugins\/leedUpdateSource\/leedUpdateSource.plugin.disabled.php":false,"\/data\/www\/rss.neurozone.fr\/plugins\/shaarleed\/shaarleed.plugin.disabled.php":true,"\/data\/www\/rss.neurozone.fr\/plugins\/urlclean\/urlclean.plugin.disabled.php":false,"\/data\/www\/rss.neurozone.fr\/plugins\/leedHomeLink\/leedHomeLink.plugin.disabled.php":true,"\/data\/www\/rss.neurozone.fr\/plugins\/scrollRead\/scrollRead.plugin.disabled.php":false,"\/data\/www\/rss.neurozone.fr\/plugins\/search\/search.plugin.disabled.php":true}

De plus il serait pratique de forcer les plugin à avoir une compatibilité avec la version de leed.

Exemple: nous sommes en version 1.8, mais certains plugins ne semblent pas compatible.
il faudrait rajouter une vérification de la version du plugin avec la version de leed

**Comment by Tabrisange** [comment]
Quitte à faire quelque chose j'aurais plutôt vu ça : 

{
	"DeleteTheCache": {enabled: true},
	"smartfeedsupdate": {enabled: false},
	"leedUpdateSource": {enabled: false},
	"shaarleed": {enabled: true},
	"urlclean": {enabled: false},
	"leedHomeLink": {enabled: true},
	"scrollRead": {enabled: false},
	"search":{enabled: true}
}

Vu qu'on sait que les plugins sont dans le dossier plugins...

**Comment by Sbgodin** [comment]
Effectivement, contrôler la version, c'est prévu dans la v2. Nous changeons l'API d'accès aux données (mysqli -> pdo), et cela aura un impact sur les greffons. Pour le reste, on verra sur place.
